apps/governance/src/routes/proposals/test-helpers/mocks.ts

```diff
@@ -62,3 +75,34 @@ export const lastWeek = new Date(-oneWeek);
 export const nextWeek = new Date(oneWeek);
 export const lastMonth = new Date(-oneMonth);
 export const nextMonth = new Date(oneMonth);
+
+export const createUserVoteQueryMock = (
+  proposalId: string | undefined | null,
+  value: VoteValue
+) => ({
```

Comment:
Its important to type these as if you miss a field the mock query will not resolve

```suggestion
): MockedResponse<GeneratedVoteQuery> => ({
```